src/modules/cmdpal/Exts/Microsoft.CmdPal.Ext.WindowWalker/Components/SearchController.cs

```diff
@@ -45,7 +46,7 @@ internal string SearchText
     /// <summary>
     /// Gets the open window search results
     /// </summary>
-    internal List<SearchResult> SearchMatches => new List<SearchResult>(searchMatches ?? new List<SearchResult>()).OrderByDescending(x => x.Score).ToList();
+    internal List<SearchResult> SearchMatches => new List<SearchResult>(searchMatches ?? []).OrderByDescending(x => x.Score).ToList();
```

Comment:
Is this just like a VS preference now?

Reply:
Something like that. The code formatter setting I have is very aggressive about it (et. al, as you'll see)